### ISSUE TITLE
Issue with Stream note

### ISSUE THREAD

**Issue by rabiibrahimi** [espocrm/espocrm#2498]
**Describe the bug**
When you have a long text stream note and using markdown syntax like code lines at first when the stream note is saved everything looks ok and the (see more) option would appear but after refreshing the page the stream note won't display properly and the option to (see more) disappears. 

**To Reproduce**
1. go to any account record => stream and type in a long text on the stream and may be add some markdown syntax and save the stream note
2. check t=out the stream note after saving you will see the (see more) option which would allow to expand the note.
3. refresh the page and visit the stream on the same account record you will notice that the stream note is not displaying correctly and that it has been cut off and also the (see more) option is not there anymore. 
...

**Expected behavior**
i expect that the stream note will keep its format after refreshing the page. 

**Screenshots**

**EspoCRM version**
version 7.2.7
<img width="955" alt="Screenshot 2022-11-09 at 15 46 56" src="https://user-images.githubusercontent.com/19490674/200878415-cc6b2f60-0673-4162-ae44-6beecc0dffc4.png">
<img width="955" alt="Screenshot 2022-11-09 at 15 48 09" src="https://user-images.githubusercontent.com/19490674/200878430-ac62d27f-fe9d-46b4-97bf-9651816fba2c.png">

**Comment by yurikuzn** [comment]
We have long texts in posts in our portal every day, I've never noticed this problem. Could not reproduce.

Bust with other stuff. Please try to investigate by yourself if you can.

**Comment by rabiibrahimi** [comment]
sure i will try and investigate and let you know. 

Thanks

**Comment by rabiibrahimi** [comment]
you are right seems this issue happened only when the stream is added to the tabs. once take it out of the tabs the stream notes are good and everything works fine. so definitely this issue is produced when stream is added on tabs

**Comment by yurikuzn** [comment]
Fix: https://github.com/espocrm/espocrm/commit/4e7462566be77b36cab4b453b78d7242b319e483

**Comment by rabiibrahimi** [comment]
thanks
